deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1035,13 +1035,20 @@ def callback_detect_integrity_event(line):
     return None
 
 
-def callback_detect_registry_integrity_event(line):
+def callback_detect_registry_integrity_state_event(line):
     match = re.match(r'.*Sending integrity control message: {"component":"fim_registry","type":"state","data":(.+)}', line)
     if match:
         return json.loads(match.group(1))
     return None
 
 
+def callback_detect_registry_integrity_clear_event(line):
+    match = re.match(r'.*Sending integrity control message: {"component":"fim_registry","type":"integrity_clear".*', line)
+    if match:
+        return True
+    return None
```

Comment:
We can change this function and the one above it to use `callback_detect_integrity_event` to get the JSON out of the message and use that to make any additional verifications we want, it would look something like this:
```python
def callback_detect_registry_integrity_state_event(line):
    event = callback_detect_integrity_event(line)
    if event and event['component'] == 'fim_registry' and event['type'] == 'state':
        return event['data']
    return None


def callback_detect_registry_integrity_clear_event(line):
    event = callback_detect_integrity_event(line)
    if event and event['component'] == 'fim_registry' and event['type'] == 'integrity_clear':
        return True
    return None
```